### PR TITLE
Report for second face-to-face meeting

### PR DESCRIPTION
**Pull request by GZhang2**

### CODE REVIEW

**Comment by GZhang2** [comment]
Hilmar's edits were incorporated, but changes made in the current commit doesn't reflect those because I made some additional changes and merged the previous request.

**Comment by hlapp** [comment]
Still working on it. I'll indicate clearly when it's ready 😄

**Comment by ncellinese** [comment]
I am sure it will be very clear!!

File: _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md [hunk 1]
```diff
@@ -5,50 +5,82 @@ author: gyzhang
...
+### Qualifier specimens
 
-Towards the end of the meeting, we did some planning for future activities. We planned out a visit for Guanyang and Gaurav to Duke in December. If selected for the Phenoscape workshop, Guarav and Guanyang can combine the visit to Hilmar with the [^workshop] workshop. Additionally, we will be having our first advisory board meeting in early 2018.
+Specifiers in a phylogenetic clade definition normally identify the nodes that define the ancestor anchoring the clade, by having to be descended from or (for _branch-based_ definitions) required to not be descended from the ancestor. However, sometimes published phylogenetic clade definitions give a specifier as a _qualifier_. A specifier in this role acts in a way as a conditional self-destruction device. A qualifier does not itself take part in determining the common ancestor anchoring the clade, but serves as an enduring requirement, for example to fall within the clade. If, for example, new data and resulting phylogenetic knowledge were to put the qualifier outside of the clade, the clade definition would then become invalid from the standpoint of nomenclature[^phylocode 11.9].
```
Comment (hlapp) [inline _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md]: @ncellinese can you proofread this paragraph in particular so that it's accurate?

**Comment by hlapp** [comment]
@gaurav please cross-check article for accuracy and indicate your approval.

@ncellinese as per explicit comment, can you please cross-check the paragraph explaining the nomenclatural view on qualifiers.

@GZhang2 please proofread after the changes.

Once you all three approve we can merge.

**Comment by ncellinese** [comment]
Can somebody send me a link to these blogs? I am not on my computer, I am on my phone in meetings and want to look at this now but can’t navigate GitHub on my fossil phone. thanks!

**Comment by ncellinese** [comment]
> Also, I think we should consistently say that phyloreferences resolve to a node while specifiers match nodes. These relationships seem pretty similar at first glance, but are the result of very different mechanisms -- respectively, logical inference within the phylogeny vs the taxonomic identification of nodes using metadata. I think it's helpful to keep the verbs we use for these two processes different as a reminder that phyloreference resolution and specifier/taxonomic matching are very different processes.
> 
> 
Actually, I think the opposite is more accurate. Specifiers resolve to nodes. The phylorefrence, which is an ontoglogized phylodefinition, is a match for a node, is just an expression

File: _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md [hunk 2]
```diff
@@ -5,50 +5,82 @@ author: gyzhang
...
...phyloreference that in a given tree doesn't "match" any nodes would correspond most closely with the nomenclatural case of a phylogenetic clade definition that becomes inapplicable when its qualifying conditions aren't met anymore by the prevailing phylogenetic knowledge. How best to implement this in OWL will need to be investigated; so far only our OWL model for branch-based clade definitions (which work by exclusion) allows a phyloreference not to match, namely when a given phylogeny does not support the exclusion. One of the challenges here is due to a limitation of OWL, our chosen language for modeling data semantics, in that OWL lacks a notion of property chain length, and thus lacks property path length as an attribute that could be used in axioms. Although the benefits of using OWL, including its large supporting community and tool ecosystem, are strong, at some point the question of best suitable implementation context for phyloreferences may need revisiting. This issue also suggests that it is advisable to decouple the language in which phyloreference semantics are formally defined and modeled (OWL) from the one in which a machine reasoner might operate or be implemented.
```
[the remark this answers is not in the document]
Comment (gaurav) [reply _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md]: How about this: "How best to implement this in OWL will need to be investigated; so far only our OWL model for branch-based clade definitions (which work by exclusion) allows a phyloreference not to match, namely when a given phylogeny does not support the exclusion" --> "How best to implement this in OWL will need to be investigated; while our OWL model can be used to construct branch-based phyloreferences that fail to resolve if an internal qualifier falls outside the clade, a similar mechanism is not currently available for node-based phyloreferences. We also do not support any kind of external qualifier."

File: _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md [hunk 3]
```diff
@@ -5,50 +5,82 @@ author: gyzhang
...
+
+## Ongoing issues left pending
+
+We discussed several issues that have surfaced in our ongoing work, but whcih we had to leave unresolved pending further research and exploration. These include the following:
+
+1. Part of honing our outreach and research publishing is answering the question how Phyloref differs from other taxonomic resolution/reconciliation approaches. How can we most succinctly articulate what Phyloref can do that other services cannot? Relatedly, how do we most persuasively communicate about what biologists need to do that we might be able to provide, in contrast to or better than other approaches or services?
+2. In light of some recent uncertainties about OWL-DL expressivity being capable of capturing all phyloreference definitions encountered in the wild, how do we best drive towards verifying that OWL remains the best formal logics language for implementing the machine-readable clade semantics part?
```
Comment (gaurav) [inline _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md]: Is "best formal logics language" the correct term here? Maybe "best knowledge representation language"?

File: _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md [hunk 4]
```diff
@@ -5,50 +5,82 @@ author: gyzhang
...
 
-![Phyloreference specifications - multiple specifiers and OTUs]({{ site.url }}/images/2ndmeeting.whiteboard2.JPG)
+![Phyloreference specifications - multiple specifiers and OTUs]({{ site.url }}/images/2017-10-project-meeting/2ndmeeting.whiteboard2.JPG)
 
-Gaurav demonstrated using [Protege] to visualize phyloref OWL files. Hilmar pointed out that these OWL files are themselves project outputs: they should be polished into clearly documented ontologies that any person familiar with OWL should be able to understand. Editing, testing and fixing the input JSON-LD files should also occur through a graphical user interface (GUI) — curators of phyloreferences should not be expected to edit JSON-LD files by hand, and a GUI could simplify the process of importing multiple phyloreferences from a single paper. This would help us expand our test suite quickly, and might form the basis of the prototype we’re aiming to develop next.
+### Qualifier specimens
```
Comment (gaurav) [inline _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md]: Should this be "Qualifying clauses"? I don't think we talk specifically about specimens in here.

Comment (GZhang2) [reply _posts/blog/2017-11-28-Second-face-to-face-project-meeting.md]: Yes, it should be qualifying clauses. The term 'qualifier' does not appear anywhere in the the current version of Phylocode.

**Comment by hlapp** [comment]
@GZhang2 I think I fixed the terminology about _qualifying clauses_ throughout. If you're in agreement, you can go ahead and merge (use "_Squash and merge_" to squash the full set of changes into a single commit; clean up the commit messages that will be enumerated to keep only the salient changes).

**Pull request merged**